addons/l10n_ec/migrations/3.4/post-migration.py

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8 -*-
+
+from odoo import api, SUPERUSER_ID
+
+def update_withhold_type(env):
+    # Recreate account tax group templates
+    ChartTemplate = env['account.chart.template']
+    tax_group_data = ChartTemplate._get_account_tax_group('ec')
+    for company in env['res.company'].search([('chart_template', '=', 'ec')]):
+        ChartTemplate = env['account.chart.template'].with_company(company)
+        ChartTemplate._load_data({'account.tax.group': tax_group_data})
```

Comment:
Just curious about the purpose of this line.

Reply:
To create the records per company.  (templates are gone in saas-16.2)